File: 3746-minimum-string-length-after-balanced-removals/3746-minimum-string-length-after-balanced-removals.js
/**
 * @param {string} s
 * @return {number}
 */
var minLengthAfterRemovals = function(s) {
    let count = 0;

    for (const ch of s) 
        count += ch === 'a' ? 1 : -1;
    
    return Math.abs(count);
};

File: 3746-minimum-string-length-after-balanced-removals/3746-minimum-string-length-after-balanced-removals.java
class Solution {
    public int minLengthAfterRemovals(String s) {
        int count = 0;
        for (int i = 0; i < s.length(); i++) 
            count += s.charAt(i) == 'a' ? 1 : -1;

        return Math.abs(count);
    }
}